src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
The only cases where this method is even used is inside special event hooks, so very few events are fired using it. I agree that there could be cases where a realistic `originalEvent` might be useful, for example when simulating Pointer events from Mouse events. But the old behavior of creating an empty object wouldn't be good enough for that case anyway.